frontend/drivers/platform_unix.c

```diff
@@ -2653,13 +2669,13 @@ static bool accessibility_speak_unix(int speed,
       /* parent process */
       speak_pid = pid;
 
-      /* Tell the system that we'll ignore the exit status of the child 
+      /* Tell the system that we'll ignore the exit status of the child
        * process.  This prevents zombie processes. */
       signal(SIGCHLD,SIG_IGN);
    }
    else
-   { 
-      /* child process: replace process with the espeak command */ 
+   {
```

Comment:
Looks like VIM didn't like these line endings. I could revert these lines as they are unrelated to the PR